ampersand-form-view.js

```diff
@@ -7,6 +7,7 @@ var result = require('amp-result');
 
 function FormView(opts) {
     opts = opts || {};
+    opts.autoRender = opts.autoRender !== undefined ? opts.autoRender : true;
```

Comment:
Still, that line is a little noodley, it wouldn't hurt to change from a ternary assignment to an if statement.